content/blog/2024/02/2024-02-23-gsoc2023-announcement.adoc

```diff
@@ -0,0 +1,95 @@
+This will be Jenkins' eighth year as a mentoring organization. 
+
+Congratulations to all link:https://summerofcode.withgoogle.com/programs/2024/organizations[195 accepted open-source organizations]!
+
+As a mentoring organization for the past six years, Jenkins has mentored 35 GSoC students by 85+ different mentors, bringing together over 110 strangers for a common idea - Jenkins! 
+
+At the heart of it, GSoC is more than just a mentoring program.
+The intention is to welcome and engage with new contributors in open source. 
+It is about giving a little of your day to make a lifetime of difference, not only for the GSoC contributors, but also for the many Jenkins users who will benefit from the improvements.  
+
+We are excited to welcome new GSoC contributors to the Jenkins family. 
+We think you will enjoy this valuable experience while developing your technical skills. 
+You will gain insights into how the community works. 
+The best part will be learning from people who are passionate about Jenkins 
+but more importantly they are passionate about wanting to make a difference for another individual (GSoC contributors) and for the betterment of the project as a whole.
+
+For detailed information on Jenkins GSoC 2023, please refer to link:/projects/gsoc/2023/[the completed projects].
+
+image:/images/gsoc/2024/GSoC2024-here-we-come.png[Here we come]
+
+== What's next?
+GSoC is link:https://opensource.googleblog.com/2024/02/mentor-organizations-announced-for.html[officially announced], and please expect more potential GSoC contributors to contact projects in our link:/projects/gsoc#contacts[Gitter and Discourse channels].
+Many communications will also happen in SIG and sub-project channels.
+We will be working hard in order to help potential participants to find interesting projects, to explore the relevant domain(s),
+and to prepare their project proposals before the deadline on April 2nd (UTC).
+Then we will process the applications, select projects, and assign mentor teams.
+
+All information about the Jenkins GSoC is available on its link:/projects/gsoc/[sub-project page].
+
+== How do I apply?
+
+Refer to the link:/projects/gsoc/students[Information for students] page for full application guidelines.
+
+We encourage interested participants to reach out to the Jenkins community early and to start exploring project ideas.
+We also encourage participants to join the weekly link:https://docs.google.com/document/d/1UykfAHpPYtSx-r_PQIRikz2QUrX1SG-ySriz20rVmE0/edit?usp=sharing[Jenkins GSoC office hours].
+These meetings are set up for participants to meet org admins and mentors, and to ask questions.
+Also, join our link:https://app.gitter.im/#/room/#jenkinsci_gsoc-sig:gitter.im[Gitter channel] and our link:https://community.jenkins.io/c/contributing/gsoc/6[Discourse server] to receive information about such incoming events in the project.
```

Comment:
```suggestion
We encourage interested participants to reach out to the Jenkins community early and start exploring project ideas.
We also encourage participants to join the weekly link:https://docs.google.com/document/d/1UykfAHpPYtSx-r_PQIRikz2QUrX1SG-ySriz20rVmE0/edit?usp=sharing[Jenkins GSoC office hours].
These meetings are for participants to meet org admins and mentors and to ask questions.
Also, be sure to join our link:https://app.gitter.im/#/room/#jenkinsci_gsoc-sig:gitter.im[Gitter channel] and link:https://community.jenkins.io/c/contributing/gsoc/6[Discourse server] to receive information about such incoming events in the project.
```
Suggestion to adjust some of the grammar and remove repeated words.
Also, it appears that the link for office hours is showing the 2023 document, is there an updated document to use at this time, or would it make sense to remove the link entirely if the office hours are not happening just yet?